### ISSUE TITLE
fetchFromURL can't get metadata from m4a file

### ISSUE THREAD

**Issue by hvianna** [Borewit/music-metadata-browser#26]
I'm getting the following error when trying to read metadata from m4a files with `fetchFromURL()`:

```
TypeError: 'cancel' can't be called on a locked stream.
```

It works fine with mp3 and flac files though.

`parseBlob()` does work with the same m4a files.

Just updated to version 1.2.0, but the problem persists.

Thanks!

**Comment by Borewit** [comment]
Which browser are using?

Do you get the error as well if you drag the m4a in the [audio-tag-analyzer](https://audio-tag-analyzer.netlify.com/)?

**Comment by hvianna** [comment]
I'm using Firefox. I just tried with Chrome and it works fine. Audio-tag-analyzer also works for me on Firefox. 
Maybe it's something on my code then, I'll try to narrow it down. Thanks!

**Comment by Borewit** [comment]
It maybe related to this issue: Borewit/readable-web-to-node-stream#1

Parsing an M4A will read result in reading the entire stream, FLAC & MP3 in most cases not.

**Comment by Borewit** [comment]
I managed to reproduce it, if you drag URL (no file) into the [audio-tag-analyzer](https://audio-tag-analyzer.netlify.com/) it will show the error.

**Comment by hvianna** [comment]
Ah nice! I didn't think of dragging the URL.

In the meantime I wrote a very simple test to try and isolate the problem:

```
const mm = require('music-metadata-browser');

mm.fetchFromUrl( 'aitai.m4a', { duration: false, skipCovers: true } )
	.then( metadata => {
		console.log( metadata );
	});
```

Using a simple web server with express:

```
const path = require('path');
const express = require('express')

const server = express()
server.use( express.static( path.join( __dirname, 'public' ) ) )
server.listen( 8000, 'localhost', () => {
	console.log( '\nServer listening on port 8000!' )
	console.log( '\n\nPress Ctrl+C to exit.' )
})
```

Works on Chrome. Same error on Firefox.

Thanks again for the prompt replies!

**Comment by Borewit** [comment]
Should be solved in [v1.2.1](https://github.com/Borewit/music-metadata-browser/releases/tag/v1.2.1)
